pkgs/by-name/tu/turbo-unwrapped/package.nix

```
@@ -63,20 +63,20 @@ rustPlatform.buildRustPackage rec {
     updateScript = nix-update-script {
       extraArgs = [
         "--version-regex"
-        "'v(\\d+\\.\\d+\\.\\d+)'"
+        "v(\\d+\\.\\d+\\.\\d+)$"
       ];
     };
   };
 
   meta = {
     description = "High-performance build system for JavaScript and TypeScript codebases";
     homepage = "https://turbo.build/";
-    changelog = "https://github.com/vercel/turbo/releases/tag/v${version}";
+    changelog = "https://github.com/vercel/turborepo/releases/tag/${toString finalAttrs.src.tag}";
```

```suggestion
    changelog = "https://github.com/vercel/turborepo/releases/tag/${finalAttrs.src.tag}";
```
Why was it here originally? `tag` should already be a string.

---

If `src` is overridden `tag` may be null, so `toString` avoids an evaluation error

---

it'd be good to have an explanatory comment, to prevent that `toString` from being removed

---

now that i think about it, referring to src seems wrong. Why not use `finalAttrs.version` directly?